src/Symfony/Component/PasswordHasher/Tests/Hasher/SodiumPasswordHasherTest.php

```diff
@@ -28,65 +28,64 @@ protected function setUp(): void
     public function testValidation()
     {
         $hasher = new SodiumPasswordHasher();
-        $result = $hasher->hash('password', null);
-        $this->assertTrue($hasher->verify($result, 'password', null));
-        $this->assertFalse($hasher->verify($result, 'anotherPassword', null));
-        $this->assertFalse($hasher->verify($result, '', null));
+        $result = $hasher->hash('password');
+        $this->assertTrue($hasher->verify($result, 'password'));
+        $this->assertFalse($hasher->verify($result, 'anotherPassword'));
+        $this->assertFalse($hasher->verify($result, ''));
```

Comment:
@wouter @chalasr  verify() has no third parameter and hash() has no second salt parameter (anymore?)